test/dotnet.Tests/CommandTests/Project/Convert/DotnetProjectConvertTests.cs

```diff
@@ -670,6 +669,44 @@ public void Directives()
                 """);
     }
 
+    /// <summary>
+    /// There should be only one <c>PropertyGroup</c> element when the default properties are overridden.
+    /// </summary>
+    [Fact]
+    public void Directives_AllDefaultOverridden()
+    {
+        VerifyConversion(
+            inputCSharp: """
+                #!/program
+                #:sdk Microsoft.NET.Web.Sdk
+                #:property OutputType=Exe
+                #:property TargetFramework=net11.0
```

Comment:
`main` for the SDK is not `net11` yet.

Reply:
I know, this is supposed to be a different version for testing, I can probably use something like `net472` to make that clearer.